de_web_plugin.cpp

```diff
@@ -2694,13 +2694,17 @@ void DeRestPluginPrivate::addSensorNode(const deCONZ::Node *node)
 
                 case ANALOG_INPUT_CLUSTER_ID:
                 {
-                    fpSwitch.inClusters.push_back(ci->id());
+                    if (modelId == QLatin1String("lumi.sensor_cube")) {
```

Comment:
We may need to extend this whitelist in future, I'm not sure but I think there were other sensors using the analog/multi-state clusters too.

Reply:
I double-checked that I introduced these clusters when adding support for the Smart Cube.  Also, the handling of the attribute values is specific to the cube.

Reply:
Alternatively we could to blacklist the “funny” sensors (so far I’ve seen reports of lumi.sensor_ht and lumi.switch_86sw2 showing additional endpoints with these clusters).